### PR TITLE
Bump Mesos dep to 1.0.0-rc2

### PR DESCRIPTION
**Pull request by gkleiman**

### CODE REVIEW

**Comment by aquamatthias** [comment]
Some methods have been deprecated.

**Comment by gkleiman** [comment]
@aquamatthias I'll look into this with @jdef's help.

**Comment by gkleiman** [comment]
Mesos v1.1 will ship with https://reviews.apache.org/r/49958/, which should let us use the new non-deprecated method without a workaround

In the meantime I updated the build definition in order to allow calls to deprecated methods.

File: project/build.scala [hunk 1]
```diff
@@ -306,7 +306,7 @@ object Dependency {
     // runtime deps versions
     val Chaos = "0.8.7"
     val Guava = "19.0"
-    val MesosUtils = "1.0.0-rc1"
+    val MesosUtils = "1.0.0-rc2"
```
Comment (aquamatthias) [inline project/build.scala]: Please add a comment here (regarding deprecation) as well, so we get reminded if we change the version here.

**Comment by aquamatthias** [comment]
LGTM

**Comment by gkleiman** [comment]
@aquamatthias done

**Comment by aquamatthias** [comment]
Waiting for tests to pass

**Pull request merged**